betty/assertion/__init__.py

```diff
@@ -502,3 +505,21 @@ def _assert_setattr(value: Any) -> Any:
         return getattr(instance, attr_name)
 
     return AssertionChain(_assert_setattr)
+
+
+_SizedT = TypeVar("_SizedT", bound=Sized)
+
+
+def assert_len_min(minimum: int = 0) -> AssertionChain[_SizedT, _SizedT]:
```

Comment:
If we can't make the types work, maybe add assert_passthrough() which takes extra Assertions to call but ignores their return values